src/index.ts

```diff
@@ -97,6 +97,92 @@ function setupOpenMRS() {
             ),
           { featureName: "vitals-biometric-form", moduleName }
         )
+      },
```

Comment:
@samuelmale these items should be read from a configuration not just hard coded into the app.

Reply:
@jonathandick They are configurable by virtue of being extensions. This all seems correct to me.